rules/helpers.go

```diff
@@ -153,12 +153,12 @@ func TableLinkForExpression(expr string) string {
 	// See also:
 	// http://stackoverflow.com/questions/1634271/url-encoding-the-space-character-or-20.
 	urlData := url.QueryEscape(fmt.Sprintf(`[{"expr":%q,"tab":1}]`, expr))
-	return fmt.Sprintf("/graph#%s", strings.Replace(urlData, "+", "%20", -1))
+	return fmt.Sprintf("graph#%s", strings.Replace(urlData, "+", "%20", -1))
```

Comment:
I don't think this will work, console templates could be any number of directories deep.